main/networking/Runtime.ts

```diff
@@ -316,16 +542,16 @@ class UDPConn {
       );
     }
 
-    const message = protos.UserInputs.encode({inputs: data}).finish();
-    let port = UDP_SEND_PORT;
-    let ip = runtimeIP;
-    // This is temporary. This will need to be changed once Runtime figures out how to send UDP data over TCP ngrok connection
-    if (defaults.NGROK && runtimeIP.includes(':')) {
-      const split = runtimeIP.split(':');
-      ip = split[0];
-      port = Number(split[1]); 
-    }
-    this.socket.send(message, port, ip);
+    const message = protos.UserInputs.encode({ inputs: data }).finish();
+    // let port = UDP_SEND_PORT;
+    // let ip = runtimeIP;
+    // // This is temporary. This will need to be changed once Runtime figures out how to send UDP data over TCP ngrok connection
+    // if (defaults.NGROK && runtimeIP.includes(':')) {
+    //   const split = runtimeIP.split(':');
+    //   ip = split[0];
+    //   port = Number(split[1]);
+    // }
```

Comment:
Haven't tested sending yet but with the UDPForwarder above, I believe we can probably remove this

Reply:
You can remove it from the normal UDPConn yea. But TCPConn and UDPTunneledConn will need different IPs unless we use paid ngrok

Reply:
Cool, will remove